patterns/2-structured/discover-your-innersource.md

```diff
@@ -83,4 +67,23 @@ TBD
 
 ## Acknowledgements
 
-* Contributions from Russ Rutledge, Ofer Hermoni and Robert Hanmer
+* Russ Rutledge
+* Ofer Hermoni
+* Robert Hanmer
+
+## Alias
+
+* Not looking for stuff internally
+* Don't bother looking
+* Find it Inside
+
+## Discussion on
```

Comment:
@spier move this discussion section here and link to it from this PR?

https://github.com/InnerSourceCommons/InnerSourcePatterns/discussions

Reply:
As these discussions are almost 8 years old, I doubt that they still have any traction. I would not create a new GitHub discussion for them.

Instead I would:
- port individual bullet points into other sections of this pattern if applicable.
- delete the rest

Reply:
Refactored content from discussions and removed the section.